libass/ass_shaper.c

```diff
@@ -682,9 +691,19 @@ static bool shape_harfbuzz(ASS_Shaper *shaper, GlyphInfo *glyphs, size_t len)
                 level == shaper->emblevels[i + 1])
```

Comment:
Unrelated to your patch: It seems to me the `return false` 6 lines prior leaks memory as it doesn't destroy `buf`, doesn't it? *(reminding me of #498 which this appears to be a part of but not all of it)*

Reply:
Yes, looks likely! Nice catch; I’ll make a fix.

Reply:
I’m going through our HarfBuzz code and fixing memory checks and other leaks like this. I may end up touching this line, so I think it’s best to split that into a follow-up PR if this one gets merged soon.